packages/serverpod_shared/test_integration/assets/load_passwords/runner_missing_passwords.dart

```diff
@@ -0,0 +1,10 @@
+import 'dart:io';
+import 'package:serverpod_shared/serverpod_shared.dart';
+
+void main() {
+  var passwords = PasswordManager(runMode: 'development').loadPasswords(
+    './config/missing_passwords.yaml',
```

Comment:
Suggestion: use a generated uuid name for the file to ensure it doesn't exist.

This test will start failing if an a file is created called this.